src/libexpr/eval.cc

```diff
@@ -2583,6 +2584,23 @@ Strings EvalSettings::getDefaultNixPath()
     return res;
 }
 
+bool EvalSettings::isPseudoUrl(std::string_view s)
```

Comment:
Any reason for moving that out of `filetransfer.hh`?

Reply:
Yes, these "URLs" are very specific to the Nix CLI (e.g. the `channel:` and `flake:` schema) and are not used by `filetransfer.cc`.